design-documents/graph-ql/coverage/add-items-to-cart/AddSimpleProductToCart.graphqls

```diff
@@ -0,0 +1,79 @@
+type Mutation {
+    addSimpleProductsToCart(input: AddSimpleProductsToCartInput): AddSimpleProductsToCartOutput
+}
+
+input AddSimpleProductsToCartInput {
+    cartId: String!
+    cartItems: [SimpleProductCartItemInput!]!
+}
+
+input SimpleProductCartItemInput {
+    sku: String!
+    quantity: Float!
+    customizable_options:[CustomizableOptionInput!]
+}
+
+input CustomizableOptionInput {
+    id: Int!
+    value: String!
+}
+
+type AddSimpleProductsToCartOutput {
+    cart: Cart
+}
+
+type Cart {
+    id: String
+    items: [CartItemInterface]
+}
+
+interface CartItemInterface @typeResolver(class: "Magento\\CatalogCheckoutGraphQl\\Model\\CartItemInterfaceTypeResolverComposite") {
+    id: Int!
+    qty: Float!
+    product: CartItemProduct!
+    image: CartItemImage!
+    prices: CartItemPrices!
+}
+
+type CartItemImage {
+    alt_text: String!
+    thumbnail: ProductMediaGalleryEntriesContent!
```

Comment:
Probably out of scope the current PR scope but would be really nice to have the absolute url here.